server/manager/src/main/java/org/apache/accumulo/manager/upgrade/Upgrader9to10.java

```diff
@@ -142,6 +148,8 @@ public void upgradeMetadata(ServerContext ctx) {
     upgradeRelativePaths(ctx, Ample.DataLevel.USER);
     upgradeDirColumns(ctx, Ample.DataLevel.USER);
     upgradeFileDeletes(ctx, Ample.DataLevel.USER);
+    // special case where old files need to be deleted
```

Comment:
Did you consider calling this new method in upgradeZookeeper instead?  I think that is called before the root tablet is loaded, which would allow deleting any old sorted logs the root tablet may reference.

Reply:
No, I didn't think any tables were loaded until after the Upgrader was finished. If that is the case, then moving it to `upgradeZookeeper` would be better.

Reply:
I think the upgrade methods are run as follows.

 * `upgradeZookeeper()` is run before the root tablet is loaded.
 * `upgradeRoot()` is run after the root tablet is loaded and before the metadata table is loaded.  
 * `upgradeMetadata` is run after the metadata table is loaded and before loading user tablets.

Reply:
Ah OK. I will move it to `upgradeZookeeper()`.